internal/agent/sender/sender.go

```diff
@@ -341,3 +485,14 @@ func (s *sender) encryptOAEPChunks(data []byte) ([]byte, error) {
 
 	return encrypted, nil
 }
+
+func (s *sender) getOutboundIP() {
+	conn, err := net.Dial("udp", "8.8.8.8:80")
+	if err != nil {
+		log.Fatalf("failed to get machine IP: %v", err)
```

Comment:
```suggestion
		return err
```
Если есть необходимость уронить по Fatal, делать это надо в одной, самой верхней точке.